components/bl-pdfViewer-component/src/index.js

```diff
@@ -0,0 +1,90 @@
+import { Document, Page } from './react-pdf.min.js';
+import { useState, useEffect, useRef } from 'react';
+import { Controls } from './subcomponents';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const sizeVariants = {
```

Comment:
Could we add properties to set custom size?

Reply:
Then the page will add or subtract a few pixels to the specified parameters, and the control block may fall out of the parent block

Reply:
Not clearly understand why we could make control block/buttons flexible.